src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1307,13 +1317,13 @@ private Task<INetworkInterface> AssociateNicWithNetworkSecurityGroupAsync(INetwo
                 () => networkInterface.Update().WithExistingNetworkSecurityGroup(networkSecurityGroup).ApplyAsync()
             );
 
-        private Task CreatePostgreSqlDatabaseUser(ConnectionInfo sshConnectionInfo)
+        private Task CreatePostgreSqlCromwellDatabaseUser(ConnectionInfo sshConnectionInfo)
```

Comment:
@tonybendis do you have any thoughts on whether we'll need to do this or something equivalent for the TES db as well? I noticed cosmosdb doesn't seem to have a user/password so wasn't sure what I should do for setup.

Reply:
Yes, you would have to create TES user and grant privileges just like we do for Cromwell. In CosmosDB case, TES has contributor role on CosmosDb so it can create the database and get the credentials. There is no such thing in Postges, so Deployer has to create DB, user and privileges, so TES can then use those credentials to do the rest (create tables and use them). 

This PR should also include logic to not create CosmosDB if creating Postgres database, and logic to not fail if CosmosDbAccountName is empty during update. If we just enable the Postgres switch, we'll end up with both CosmosDB and Postgres DB. So we'd have to revisit the deployer again beyond just enabling the Postgres switch.

Postgres FQDN server name, user and PW should be included in docker-compose.yml (in the environment variables section just like the others).
Postgres environment variables should be empty if using Cosmos and CosmosDbAccountName should be empty is using Postgres. That way, TES will know which repository implementation to use.

Reply:
I realize github didn't chain this message so reposting on this thread- 
I made some of your changes but disagree with including the switch logic in this PR. There's a separate issue for actually hooking up TES from cosmos to postgres and in my opinion that should include the Update logic (I made a note in https://github.com/microsoft/CromwellOnAzure/issues/386 to make that explicit). I think including the "dont do cosmos" logic before we actually have TES hooked up to postgres is premature.

Reply:
OK